plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteClient.java

```diff
@@ -506,7 +506,7 @@ public Map<String, Object> getTableProperties(Connection connection, JdbcTableHa
         String schemaName = requireNonNull(schemaTableName.getSchemaName(), "Ignite schema name can not be null").toUpperCase(ENGLISH);
         String tableName = requireNonNull(schemaTableName.getTableName(), "Ignite table name can not be null").toUpperCase(ENGLISH);
         // Get primary keys from 'sys.indexes' because DatabaseMetaData.getPrimaryKeys doesn't work well while table being concurrent modified
```

Comment:
Is this comment still valid even after upgrading the server version?

Reply:
Yes, tested on my local, the `DatabaseMetaData.getPrimaryKeys` would return empty primary keys sometimes.